nginx-mixin/dashboards/nginx-metrics.json

```diff
@@ -0,0 +1,567 @@
+{
+  "__inputs": [
+    {
+      "description": "",
+      "label": "Prometheus",
+      "name": "DS_PROMETHEUS",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus",
+      "type": "datasource"
+    }
+  ],
+  "__requires": [
+    {
+      "id": "grafana",
+      "name": "Grafana",
+      "type": "grafana",
+      "version": "5.0.0"
```

Comment:
Wait actually I just realised, this is a dashboard built on the much much older Grafana scheme, we'll need to upgrade this at the very least to Grafana 10/11 to remove any Angular based panels